### PR TITLE
Doc: Generate model from third-party-resources

### PR DESCRIPTION
**Pull request by yue9944882**
/cc @brendandburns

### CODE REVIEW

File: docs/generate-model-from-third-party-resources.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+Note that if you can't find a 1.15+ cluster anywhere, consider run one locally by [kind](https://github.com/bsycorp/kind) then
+don't forget to deploy your CRDs into the local-run kubernetes cluster.
+
+
+##### Generate Java model from the downloaded openapi spec
+
+Run the following command and there will be a java sources generated under `/tmp/java`. 
+
+```
+docker run -i --rm yue9944882/java-model-gen < /tmp/swagger | tar -xzf - -C /tmp/
```
Comment (yue9944882) [inline docs/generate-model-from-third-party-resources.md]: i don't have access to the offical registry so i pushed them my registry temporarily

Comment (brendandburns) [reply docs/generate-model-from-third-party-resources.md]: Neither do I (afaik)

Comment (aespinosa) [reply docs/generate-model-from-third-party-resources.md]: Is there a reference `Dockerfile` on how this image was generated?

**Pull request merged**